release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -259,6 +299,36 @@ SCTP is a reliable message based protocol that runs on top of an IP network.
 When enabled, you can use SCTP as a protocol with both Pods and Services.
 For more information, see xref:../networking/using-sctp.adoc#using-sctp[Using SCTP].
 
+[id="ocp-4-4-using-dns-forwarding"]
+==== Using DNS forwarding
+
+You can use DNS forwarding to override the forwarding default configuration on a
+per-zone basis by specifying which name server should be used for a given zone.
```

Comment:
Is "which name" correct?

Reply:
I believe so. The DNS forwarding docs reference "name servers" several times. Google also confirms it's a networking term.